### ISSUE TITLE
deployment of downloaded web modeler xml not possible

### ISSUE THREAD

**Issue by MainAero** [camunda/zeebe#10726]
**Describe the bug**

Context:
- model source xml files are managed in git repositories
- deployment is done using CI and `zbctl` automatically after creating a release

Modeling using the new connector feature is currently (as far as we know) only possible in the new web modeler. Following our process of CI deployment we downloaded the model using the BPMN 2.0 XML download option of the web modeler.

Deployment using the latest `zbctl` version fails with:
```
Error: rpc error: code = InvalidArgument desc = Command 'CREATE' rejected with code 'INVALID_ARGUMENT': Expected to deploy new resources, but encountered the following errors:
'model.bpmn': SAXException while parsing input stream
```

We tried to open it in the latest desktop modeler as well (hope was this modeler will "convert" it somehow) without luck. The desktop modeler removes all tasks with connectors silently.

**To Reproduce**

Create a diagram in web modeler which uses a task with connector feature. Download it and try to deploy it using `zbctl`.

**Expected behavior**

Deployment is done.

**Comment by oleschoenburg** [comment]
Could you share a model that we can use to reproduce this?

**Comment by pihme** [comment]
See https://github.com/camunda/web-modeler/issues/2646 for potential causes to file corruption 

TL;DR: There were some bugs in web modeler where the XML was corrupted in the following way: 
`&#34;` was used instead of `"` 

The bugs are fixed in Web Modeler now. 

Workaround would be to replace `&#34;` with `"`

**Comment by menski** [comment]
This should be fixed in the web modeler, please let us know if you still encounter issues.
